Add affiliation results (#87)
greenelab/iscb-diversity-manuscript: content/10.methods.md

When labeling affiliations with countries, we only used these values when geotext did not return results or had ambiguity amongst countries without multiple matches.
For more details on this approach, consult the accompanying [notebook](https://github.com/greenelab/iscb-diversity/blob/5213ba3451520af3967f74d8f58553dade0a826c/07.affiliations-to-countries.ipynb) and [label dataset](https://github.com/greenelab/iscb-diversity/blob/5213ba3451520af3967f74d8f58553dade0a826c/data/affiliations/geocode.jsonl).

For ISCB honorees, during the curation process, if an honoree was listed with their affiliation at the time, we recorded this affiliation for analysis.
For ISCB Fellows, we used the affiliation listed on the ISCB page.
Because we could not find affiliations for the 1997 and 1998 RECOMB keynote speakers' listed for these years, they were left blank.

> If an author or speaker has more than one affiliation, each is inversely weighted by the number of affiliations that individual has.

Reviewer: ```suggestion
If an author or speaker had more than one affiliation, each was inversely weighted by the number of affiliations that individual had.
```

### Affiliation Analysis

> For each country, we computed the expected number of honorees by multiplying the proportion of authors whose affiliations are in that country with the total number of honorees.

Reviewer: ```suggestion
For each country, we computed the expected number of honorees by multiplying the proportion of authors whose affiliations were in that country with the total number of honorees.
```

> We also performed an enrichment analysis to examine the difference in country affiliation proportions between ISCB honorees and Pubmed corresponding authors.

Reviewer 2: ```suggestion
We then performed an enrichment analysis to examine the difference in country affiliation proportions between ISCB honorees and field-specific corresponding authors.
```

We calculated each country's enrichment by dividing the observed proportion of honorees by the expected proportion of honorees.

> The variance of the log~2~ enrichment is estimated using the delta method with a small continuity correction to avoid dividing by 0.

Reviewer: ```suggestion
The variance of the log~2~ enrichment was estimated using the delta method with a small continuity correction to avoid dividing by 0.
```

> The variance of the log~2~ enrichment was estimated using the delta method with a small continuity correction to avoid dividing by 0.

Reviewer 2: Is there a citation we could provide that would provide more info on this for readers? Our source code is probably more helpful than a citaiton, but if there is a good reference that could be nice.

Author: Good idea. I'll add this book `[@isbn:978-0849394447]`.
